packages/plugin-legacy/tsdown.config.ts

```diff
@@ -1,8 +1,71 @@
+import path from 'node:path'
+import fs from 'node:fs'
 import { defineConfig } from 'tsdown'
+import { parseTarGzip } from 'nanotar'
+
+// for rolldown-vite v7.0.9, check https://pkg.pr.new/~/vitejs/rolldown-vite
+const PLUGIN_LEGACY_FOR_ROLLDOWN_VITE =
+  'https://pkg.pr.new/vitejs/rolldown-vite/@vitejs/plugin-legacy@5c52bbc'
```

Comment:
How about adding this to devDependencies and copy it from `node_modules`? That seems more robust than fetching from pkg.pr.new on every build.

Reply:
That sounds definitely better. I'll update the PR.

Reply:
Updated 👍